expcore/Mini_games.lua

```diff
@@ -100,73 +108,117 @@ function Mini_games.get_running_game()
     return started_game[1]
 end
 function Mini_games.start_game(name,parse_args)
-    local mini_game = Mini_games.mini_games[name]
-    if mini_game == nil then
-        return "This mini_game does not exsit"
-    end
-
-    if parse_args then
-        if  mini_game.options ~= #parse_args then
-            return "Wrong number of arguments"
+    game.print(parse_args)
+    game.print(vars.is_lobby )
+    if vars.is_lobby == true  or vars.is_lobby =="true" then
```

Comment:
Why would you allow setting a string for a boolean value.

Reply:
i got some incosities so i just put that in

Reply:
I would really try to remove this as you shouldnt have a string for a boolean as it may lead to confussing bugs. For example the most common way to test a boolean is `if boolean then` but because it might to a string `"false"` would return as a truthy value.

Reply:
i gue sill change /set_lobby